array-methods-task2.js

```diff
@@ -0,0 +1,20 @@
+// Filter range
+// importance: 4
+
+// Write a function filterRange(arr, a, b) that gets an array arr, looks for elements with values higher or equal to a and lower or equal to b and return a result as an array.
+
+// The function should not modify the array. It should return the new array.
+
+// For instance:
+
+// let arr = [5, 3, 8, 1];
+
+// let filtered = filterRange(arr, 1, 4);
+
+// alert( filtered ); // 3,1 (matching values)
+
+// alert( arr ); // 5,3,8,1 (not modified)
+function filterRange(arr, a, b) {
+  let resultArray = arr.filter((item) => item >= a && item <= b);
+  return resultArray;
```

Comment:
Creating a new variable just to return it is unecessary, you can do:

```suggestion
  return arr.filter((item) => item >= a && item <= b);
```

Reply:
I did think about it. Thanks for the reminder.